docs/FAQs.md

```diff
@@ -19,3 +19,11 @@ As this is an evolving product, this requirement may be adjusted in the future,
 - We use JavaScript to collect data to help us improve Test Pilot in accordance with our [Terms of Use](https://testpilot.firefox.com/terms) and [Privacy Notice](https://testpilot.firefox.com/privacy).
 
 Since Test Pilot is an open source project, you can see all of the cool ways we use JavaScript by [examining our code](https://github.com/mozilla/testpilot/).
+
+### I tried to enable an experiment, but it didn't work. What went wrong?
+
+Well it count be a number of things. Test Pilot experiments are just add-ons, so if you've done anything to your copy of Firefox to change the way add-ons install, that...
```

Comment:
typo: "count" > "could".

Also, not sure if the lorem ipsum is useful below or if we want to remove this FAQ until we have a fuller answer.